src/Nethermind/Nethermind.Shutter/Contracts/ValidatorRegistryContract.cs

```diff
@@ -45,7 +45,14 @@ public bool IsRegistered(BlockHeader header, in Dictionary<ulong, byte[]> valida
         for (uint i = 0; i < updates; i++)
         {
             Update update = GetUpdate(header, updates - i - 1);
-            Message msg = new(update.Message.AsSpan()[..46]);
+
+            if (update.Message.Length != 46 || update.Signature.Length != 96)
```

Comment:
Which message is 96 bytes long ? 
I can only see the 46 byte message in https://github.com/gnosischain/specs/blob/master/shutter/low-level.md#validator-registry

Reply:
It's a BLS signature (compressed G2 point) which must be 96 bytes, an exception will be thrown [here](https://github.com/NethermindEth/nethermind/blob/master/src/Nethermind/Nethermind.Crypto/BlsSigner.cs#L63) if it's not the correct length